tests/hpc/slurm_master_backup_db.pm

```diff
@@ -0,0 +1,69 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017-2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: HPC_Module: slurm failover - database
+#    This test is setting up slurm control backup nodes with accounting
+#    configured (database) so that the failover of the ctls can be tested
+# Maintainer: Sebastian Chlad <sebastian.chlad@suse.com>
+
+use base "hpcbase";
+use strict;
+use warnings;
+use testapi;
+use lockapi;
+use utils;
+
+sub run {
+    my $self  = shift;
+    my $nodes = get_required_var("CLUSTER_NODES");
+    $self->prepare_user_and_group();
+
+    zypper_call('in nfs-client rpcbind');
+    systemctl 'start nfs';
+    systemctl 'start rpcbind';
+    assert_script_run("showmount -e 10.0.2.1");
+    assert_script_run("mkdir -p /shared/slurm");
+    assert_script_run("chown -Rcv slurm:slurm /shared/slurm");
+    assert_script_run("mount -t nfs -o nfsvers=3 10.0.2.1:/nfs/shared /shared/slurm");
+
+    # provision HPC cluster, so the proper rpms are installed
+    # and proper services are enabled and started
+    zypper_call('in slurm slurm-munge slurm-slurmdbd');
```

Comment:
maybe we can move this call to the first one in line 27, so if something happens with package installation, you don't need to run extra commands.

Reply:
Hi @jlausuch I kept the former separation in the code, so that the HPC specific stuff is being zypper in later in the code. The reason being - I plan to abstract certain stuff even more to the specific hpc libs.
But If you still your comment holds and I do not convinced you, I can always change it as you asked for.

Reply:
It's ok if you have a good reason for that. Normally, all the packages installation shall be done at the beginning before anything else to save time in case of package installation problems.